### ISSUE TITLE
Generate output files on SIGTERM?

### ISSUE THREAD

**Issue by matthiasotto** [ccampo133/Plot-Pings-in-Python#15]
I am running this on my Pi to help track down ISP issues. I want to push it as a job with `&` and then `disown` it so my SSH connection doesn't affect the job when I disconnect. 

Later, I might want the plot file to be written by sending `SIGTERM` to the process.

Would this make sense?

**Comment by matthiasotto** [comment]
Furthermore, this would allow to run it on a CRON job to create a daily PNG to avoid having to draw a massive single PNG to capture a whole week. 🙂

**Comment by ccampo133** [comment]
Makes sense. I originally wrote this many years ago and realize there are substantial flaws with how it works. Probably time for a re-write!
